lib/invidious/search.py

```diff
@@ -22,31 +22,31 @@ def sortBy(sort_by="relevance"):
     return sort_by
 
 
-def newSearch(type, sort_by=None, history=False):
+def newSearch(search_type, sort_by=None, history=False):
     if (query := inputDialog(heading=30002)):
         if sort_by is None:
             sort_by = sortBy()
         if history:
-            search_history.record(type, query, sort_by)
+            search_history.record(search_type, query, sort_by)
     return (query, sort_by)
 
 
-def searchHistory(type):
+def searchHistory(search_type):
     return Queries(
-        reversed(search_history[type].values()), category=queryTypes[type]
+        reversed(search_history[search_type].values()), category=queryTypes[search_type]
     )
 
 
-def removeSearchQuery(type, query):
-    search_history.remove(type, query)
+def removeSearchQuery(search_type, query):
+    search_history.remove(search_type, query)
     containerRefresh()
 
 
 __search_url__ = f"plugin://{getAddonId()}/?action=search"
 
-def clearSearchHistory(type=None, update=False):
-    search_history.clear(type=type)
-    if type:
+def clearSearchHistory(search_type=None, update=False):
+    search_history.clear(type=search_type)
```

Comment:
shouldn't that be:
search_history.clear(search_type=search_type)

Reply:
Ahh, shoot. Good catch! Updated and pushed.